tracepoint_perf/src/input_file.rs

```diff
@@ -70,11 +70,11 @@ impl InputFile {
         return Ok(());
     }
 
-    pub fn read_struct<T>(&mut self, value: &mut T) -> io::Result<()>
+    pub(crate) fn read_struct<T>(&mut self, value: &mut T) -> io::Result<()>
```

Comment:
Note: Since we're using a crate-internal rule of "Copy implies POD", explicitly mark this method as crate-internal. It was already internal since the `InputFile` type is `pub(crate)`, but this makes it more explicit.